src/app/containers/MediaPlayer/index.jsx

```diff
@@ -17,12 +17,20 @@ import {
   emptyBlockArrayDefaultProps,
 } from '#models/propTypes';
 
-const MediaPlayerContainer = ({ blocks, placeholder }) => {
+const MediaPlayerContainer = ({ blocks, placeholder, embedOverrides }) => {
   const { id, platform, origin } = useContext(RequestContext);
   const { lang } = useContext(ServiceContext);
   const { enabled } = useToggle('mediaPlayer');
   const isAmp = platform === 'amp';
 
+  // Player settings can be overridden for other page types (eg, CPS)
+  // TODO: Refactor in https://github.com/bbc/simorgh/issues/4418
+  const type = embedOverrides.type || 'articles';
+  const assetId = embedOverrides.id || id;
+  const Wrapper = embedOverrides.wrapper || GridItemConstrainedMedium;
+  const showPlaceholder = type === 'articles' || embedOverrides.showPlaceholder;
```

Comment:
This concerns me. Firstly, if whether the media player placeholder should be used can be determined based on page type wouldn't it be more maintainable for this to be configured somewhere centrally rather than dotted around the code? 

And secondly where does the requirement for per-media-item embed overrides come from? There's nothing in this PR or the associated issue that describes what we're trying to achieve here unless I've missed it.

Finally shouldn't this relate to the work that @aeroplanejane is doing to try and harmonise requirements for player use cases?

Reply:
Thats fair, we wanted to avoid a large refactor now, but Im happy to do it now instead :)

Reply:
> This concerns me. Firstly, if whether the media player placeholder should be used can be determined based on page type wouldn't it be more maintainable for this to be configured somewhere centrally rather than dotted around the code?

It can't be determined based on page type long-term.  MAP pages need the placeholder removed; STY pages want to keep it.  Both of these are implemented through the "CPS" page.  The logic for managing this wouldn't be "dotted around the code" - it will be in the CPS page.

> And secondly where does the requirement for per-media-item embed overrides come from? There's nothing in this PR or the associated issue that describes what we're trying to achieve here unless I've missed it.

Same thing - design for MAPs needs the player to be 8 columns - STY still uses 6

Reply:
I see where you're coming from @ryanmccombe but I think unfortunately we've created this constraint for ourselves. In my mind, the user experience for STYs should be broadly consistent with other articles and MAPs like our other media pages, and the code should reflect that goal. i.e. it seems short-sighted to treat them both as instance of a 'CPS' page type, that's an implementation detail that will cease to have any meaning before long.

Reply:
Yep @jamesdonoh - this has been a bit disorganised on our part - what is now the CPS page was originally created by our pod with the intention of being the MAP page.

It got repurposed and renamed into a "CPS page" a few weeks ago.  I'm not at all sold on that - pages should map to product needs - not to the underlying data source.

This merging did let us move faster initially, but the designs we have for MAPs are very different to articles / STYs.

We probably haven't communicated that properly, so this shared page type is what we have at the minute.  Unless seperating out STYs and MAPs is part of the planned refactor happening around this work right now, we still need a way to pass props into the media player that modifies the placeholder and width

Reply:
Yep can totally see how it's come about, I don't think changing the way we handle STYs and MAPs entirely is in scope for this PR but I do think it should be looked at before we actually release anything to get ourselves on the right track for the next stage of work. Maybe one for the agenda next Mon/Tue.